deploy/custom_resources/custom_authorizer/jwt_services.py

```diff
@@ -1,46 +1,13 @@
 import os
 
 import requests
-from jose import jwk
-from jose.jwt import get_unverified_header, decode, ExpiredSignatureError, JWTError
+import jwt
+
 import logging
 
 logger = logging.getLogger()
 logger.setLevel(os.environ.get('LOG_LEVEL', 'INFO'))
 
-# Configs required to fetch public keys from JWKS
```

Comment:
We don't need this because pyJWT does it with some inbuilt method ?

Reply:
yes - https://pyjwt.readthedocs.io/en/latest/usage.html (re: `PyJWKClient()`)